bootstrap/src/contexts/Auth/backend--tiger.js

```diff
@@ -1,17 +1,76 @@
-import tigerFactory, { AnonymousAuthProvider } from "@gooddata/sdk-backend-tiger";
+import tigerFactory, {
+    ContextDeferredAuthProvider, redirectToTigerAuthentication,
+    TigerTokenAuthProvider,
+} from "@gooddata/sdk-backend-tiger";
+import { debounce, throttle } from "lodash";
```

Comment:
Nitpick: we should use the per method imports like `import throttle from "lodash/throttle";` to keep the bundle smaller (we don't check this in this repo).